### DISCUSSION TITLE
Spotify daemon not started (on BBB)

### DISCUSSION THREAD

**Discussion by fillods** [rern/rAudio#768] [Fixed Issues] [answered]
With recent updates (past one or two weeks?), rAudio fails to start the spotifyd daemon on a BeagleBoneBlack board.
NB: this is a **single** CPU board.

The daemon failed to start with an error reported by `tasket`. Indeed, there's no such 4th CPU core on a board with only one CPU.
I manged to fix the issue on my BBB by changing the file `/etc/systemd/system/spotifyd.service`  :

```diff
@@ -6,7 +6,7 @@
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/taskset -c 3 /usr/bin/spotifyd --no-daemon
+ExecStart=/usr/bin/spotifyd --no-daemon
 ExecStartPost=/srv/http/bash/cmd.sh nicespotify
 Restart=always
 RestartSec=12

``` 
Could it be the commit aa6a9ca13a6c24a0 against `install.sh` which is troublesome ?
FYI, there's no such file `/boot/kernel.img` on (my) BBB.

I haven't checked, but the `upmpdcli` service might also suffer from the same problem.

Hope this helps other owners of board with less than 4 CPU cores.

**Comment by rern** [comment, marked as answer]
Next update, the script will switch to `nproc` for CPU cores count.
By the way, please explain how to get **rAudio** running on BeagleBone boards as a new post in category **Tips**. So we can have a reference of compatible devices besides RPi.

**Reply by fillods** [reply]
Thank you for the answer.

Installating  **rAudio** on BeagleBone boards has already been described in discussion https://github.com/rern/rAudio-1/discussions/299 but it was not placed in the category **Tips**. This is now the case 🙂
